data_integration/logging/run_log.py

```diff
@@ -54,7 +54,8 @@ class SystemStatistics(Base):
     """System stats measurements"""
     __tablename__ = 'data_integration_system_statistics'
 
-    timestamp = sqlalchemy.Column(sqlalchemy.TIMESTAMP(timezone=True), primary_key=True, index=True)
+    run_id = sqlalchemy.Column(sqlalchemy.TIMESTAMP(timezone=True), primary_key=True, index=True, nullable=True)
```

Comment:
This should not be a timestamp but something which is set by the logger to the run_id as corresponding to the run_id in data_integration_run table.

Reply:
E.g. here we have to also insert the `self.run_id` https://github.com/mara/data-integration/blob/master/data_integration/logging/run_log.py#L124 and in the system statsitics thread has to be moved to the back after this line https://github.com/mara/data-integration/blob/master/data_integration/execution.py#L147

Reply:
I did the related changed.

Reply:
I did the related change.